README.md

```diff
@@ -50,14 +50,7 @@ my_builder_cronos:
         shell: /bin/bash
 ```
 
-option   | description
----------|-----------------------------------------
 key      | Unique key that wraps all the cron configured for the current application.
-mailto   | Sets the default email address for all cron output to go to.
```

Comment:
Why were these options removed from the docs?

Reply:
This looks like a bug in the PhpStorm function something like "Fix missing outter table pipes | ". Sorry i missed that.